frontend/pages/journey/new/template/[id].vue

```diff
@@ -0,0 +1,716 @@
+                                    },
+                                }"
+                                @focus="handleFocus"
+                                @hide="handleBlur"
+                                @input="handleFocus"
+                            />
+                            <br />
+                            <div class="ml-0.5 mt-1 h-3 leading-3">
+                                <span
+                                    class="text-xs text-mahagony-600 dark:font-medium dark:text-mahagony-300"
+                                    >{{ errors.journeyRange }}</span
+                                >
+                            </div>
+                            <label
+                                for="journey-range-calendar"
+                                class="pointer-events-none absolute left-0 top-0 overflow-hidden whitespace-nowrap py-4 pl-2.5 text-sm transition-all duration-300"
+                                :class="{
+                                    'text-natural-600': !isFocused,
+                                    'dark:text-natural-200': !isFocused,
+                                    'text-calypso-600': isFocused,
+                                    '-translate-x-6 -translate-y-4 scale-75':
+                                        isFocused || journeyRange,
+                                    'translate-y-0 scale-100':
+                                        !isFocused && !journeyRange,
+                                }"
+                            >
+                                <T key-name="form.input.journey.dates" />
+                            </label>
+                        </div>
+                        <div
+                            class="flex items-center text-xs text-natural-700 dark:text-natural-400"
+                        >
+                            <T key-name="template.recommended.duration" />
+                            {{ days }}
+                            <T key-name="template.days" />
+                            <button
+                                type="button"
+                                @click="isInfoDialogVisible = true"
+                            >
+                                <span
+                                    class="pi pi-info-circle pl-1"
+                                    :class="
+                                        tooShort
+                                            ? 'text-dandelion-400 dark:text-pesto-600'
+                                            : 'text-natural-500 hover:text-natural-900 dark:text-natural-400 dark:hover:text-natural-100'
+                                    "
+                                ></span>
+                            </button>
+                            <TemplateDialogsCreationDurationInfo
+                                :is-visible="isInfoDialogVisible"
+                                @close="isInfoDialogVisible = false"
+                            />
```

Comment:
place extra dialogs on the bottom of the file in an div id="dialogs" for better structure